src/subtle.ts

```diff
@@ -0,0 +1,5 @@
+import { webcrypto } from 'crypto'
+
+const isBrowser = typeof window !== 'undefined'
+
+export const subtle = isBrowser ? window.crypto.subtle : webcrypto.subtle
```

Comment:
Ah, not I get it why it all changed from `crypto.subtle` to simply `subtle`. Nice 👍